fossee_project/settings.py

```diff
@@ -94,3 +97,34 @@
 # https://docs.djangoproject.com/en/1.6/howto/static-files/
 
 STATIC_URL = '/static/'
+# Set this varable to <True> if smtp-server is not allowing to send email.
+EMAIL_USE_TLS = True
+
+EMAIL_HOST = 'smtp-auth.iitb.ac.in'
+
+EMAIL_PORT = 25
+
+EMAIL_HOST_USER = ''
+
+EMAIL_HOST_PASSWORD = ''
+
+# Set EMAIL_BACKEND to 'django.core.mail.backends.smtp.EmailBackend'
+# in production
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'                    
+
+
+# SENDER_EMAIL, REPLY_EMAIL, PRODUCTION_URL, IS_DEVELOPMENT are used in email
+# verification. Set the variables accordingly to avoid errors in production
+
+# This email id will be used as <from address> for sending emails.
+# For example no_reply@<your_organization>.in can be used.
+#SENDER_EMAIL = 'your_email'
+
+# Organisation/Indivudual Name.
+#SENDER_NAME = ''
+
+# This email id will be used by users to send their queries
+# For example queries@<your_organization>.in can be used.
+#REPLY_EMAIL = ''
+
+GOOGLE_RECAPTCHA_SECRET_KEY = '6LcDdk0UAAAAAOVRYSFOh07HwXMl0I9fPzTw4_VR'
```

Comment:
This is a secret key, need not push this on public repo.
You can put here text as '<YOUR RECAPTCHA SECRET KEY>'